PubSubSpec.md

```diff
@@ -84,6 +84,14 @@ MAPPING-ENTRY-TYPE = 206
 SEQ-NO-TYPE = 204
 ```
 
+A mapping entry MAY contain additional information such as the time of publication of the data. Any additional information MUST be encoded as one or more TLV blocks following the `ApplicationName` block. SVS-PS implementations SHOULD provide a mechanism to include additional mapping information, and SHOULD allow applications to filter incoming publications based on the recevied mapping entry. Implementations MAY also provide mechanisms for automatic handling of well-known additional blocks, such as fetching only recent data using the `Timestamp`.
```

Comment:
[nit] maybe spell "mapping entry" as `MappingEntry` to make it clear you're referring to the TLV format, although it should be obvious already...

typo: recevied -> received

Reply:
> fetching only recent data using the Timestamp

This, combined with the (re)use of TimestampNameComponent, seems to suggest that the Timestamp should be included in the Interest name as a component. Is that the intention? If not, I suggest rephrasing it, e.g.:

...*fetching only recent data based on the received `Timestamp` block*
...*fetching only recent data based on the `Timestamp` block included in the mapping entry*